src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -43,19 +44,26 @@ public static void UseDefaultCredentials_Roundtrips()
             Assert.False(cws.Options.UseDefaultCredentials);
         }
 
-        [ActiveIssue(20358, TargetFrameworkMonikers.NetFramework)]
         [ConditionalFact(nameof(WebSocketsSupported))]
         public static void SetBuffer_InvalidArgs_Throws()
         {
+            // Recreate the minimum WebSocket buffer size values from the Framework version of WebSocket,
```

Comment:
nit: "Framework" should be either ".NET Framework". You can also use "Desktop" but it's preferable to use '.NET Framework" which is the official name.